core/src/Kokkos_Crs.hpp

```diff
@@ -304,11 +304,11 @@ struct CountAndFillBase {
   Functor m_functor;
   counts_type m_counts;
   struct Count {};
-  inline void operator()(Count, size_type i) const {
+  KOKKOS_INLINE_FUNCTION void operator()(Count, size_type i) const {
```

Comment:
In the spirit of removing `inline` from functions being defined in the class declaration.
```suggestion
  KOKKOS_FUNCTION void operator()(Count, size_type i) const {
```

Reply:
I've probably missed some memo or discussion, but why is that preferable, or what's the guideline?

Reply:
Do not inline function that are defined in the body of a class

Reply:
Ok, will revise